csrc/preseg_passes/translate_repeat_to_expand.cpp

```diff
@@ -145,36 +143,26 @@ class RepeatToExpandTranslator {
 
       const auto& info = repeat_info_map_it->second;
 
-      if (info.cat_inp_tvs.size() < 2) {
+      const auto num_repetitions = (int64_t)info.cat_inp_tvs.size();
+
+      if (num_repetitions < 2) {
```

Comment:
I believe `repeat` handle num_repetitions==1 just fine in the general case. That would save us a corner case check for readability.